net-misc/mosquitto/mosquitto-1.6.9.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="An open source MQTT broker"
+DESCRIPTION="An open source MQTT brokerz"
```

Comment:
Should be different.

Reply:
Not really different....

Reply:
Would use this instead:
"Eclipse Mosquitto is an open source (EPL/EDL licensed) message broker that implements the MQTT protocol versions 5.0, 3.1.1 and 3.1. Mosquitto is lightweight and is suitable for use on all devices from low power single board computers to full servers.
The MQTT protocol provides a lightweight method of carrying out messaging using a publish/subscribe model. This makes it suitable for Internet of Things messaging such as with low power sensors or mobile devices such as phones, embedded computers or microcontrollers."

Reply:
Resolved :)